### PR TITLE
Fix CSS selectors for boolean attributes

### PR DESCRIPTION
**Pull request by distantnative**
### Fixes
- Use strict CSS selectors for boolean (data) attributes

### Breaking changes
- CSS attribute selectors must be written fully qualified (e.g. `[data-hidden="true"]` as only `[data-hidden]` can start matching also elements where that attribute is `false`)

### For review team

- [x] Add changes to release notes draft in Notion

### CODE REVIEW

File: panel/src/components/Layout/Table.vue [hunk 1]
```diff
@@ -398,7 +398,7 @@ export default {
 	color: var(--table-color-th-text);
 	background: var(--table-color-th-back);
 }
-.k-table th[data-has-button] {
```
Comment (bastianallgeier) [inline panel/src/components/Layout/Table.vue:402]: While going through all the examples, I ran into a breaking issue in the lab. If you ever use one of the attributes manually and you don't make it strict, this is basically a breaking change. I wonder how we deal with this.

Comment (distantnative) [reply panel/src/components/Layout/Table.vue:402]: What do you mean with using them manually?

Comment (bastianallgeier) [reply panel/src/components/Layout/Table.vue:402]: In the lab, we use `<th data-has-button>` in our example markup. That used to work so far, but does no longer work now.

Comment (distantnative) [reply panel/src/components/Layout/Table.vue:402]: Hmmm we could for the moment to add selectors for both. But those use cases will break sooner or later anyway when Vue renders `data-attribute="false"` ad those selectors just say "should exist at all" not whether they are true/positive.

Comment (bastianallgeier) [reply panel/src/components/Layout/Table.vue:402]: That's true. I just really wonder if we need to consider this a breaking change or not. Feels still a bit new to get used to such cases.

Comment (distantnative) [reply panel/src/components/Layout/Table.vue:402]: We could also add it to `v5/develop` instead and list it as breaking change.

Comment (bastianallgeier) [reply panel/src/components/Layout/Table.vue:402]: I think if we want to be truly carefuly, we should do it like that. It feels at least weird to do that already in 4.1, that's why I'm leaning towards that suggestion tbh.

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**